### ISSUE TITLE
Comparison with established table/grid libraries

### ISSUE THREAD

**Issue by dandv** [aslagle/reactive-table#10]
There's a lot of prior art when it comes to table/grid components. Would be interesting to know how this library compares to [DataTables](datatables.net) (check out @aldeed's [tabular](https://github.com/aldeed/meteor-tabular) package), [Dynatable](www.dynatable.com), [Tablesorter](https://github.com/Mottie/tablesorter), [SlickGrid](https://github.com/mleibman/SlickGrid) etc.
- pagination
- sorting by clicking on column headers
- filtering
- [resizable columns](https://github.com/alfajango/jquery-dynatable/issues/62)
- reorderable columns
- in-place editing
- mobile/responsive support

**UPDATE**: Here's a comparison: http://reactive-table.meteor.com/

**Comment by brylie** [comment]
As the lumapictures:meteor-jquery-datatables states, [aldeed:tabular](https://github.com/aldeed/meteor-tabular) should be a primary consideration for DataTables support. Please add aldeed:tabular to the comparison.

**Comment by dandv** [comment]
@brylie: `aldeed:tabular` wraps Datatables.net, which is listed in the comparison: http://reactive-table.meteor.com/

BTW, you can contribute to the comparison table by editing [table-features.js](https://github.com/aslagle/reactive-table/blob/master/examples/table-features/table-features.js).

**Comment by Custos** [comment]
@dandv such great work.  Any thoughts on mobile/responsive support for reactive tables?  :)

**Comment by gustawdaniel** [comment]
Link:

http://reactive-table.meteor.com/

does not works now.

**Comment by dandv** [comment]
@gustawdaniel, yes MDG removed the free hosting under .meteor.com a few years ago.

@aslagle, is the demo/comparison available anywhere else?

**Comment by aslagle** [comment]
No, I'm afraid not. I'm sure the info in it is also very outdated by now.
